.github/workflows/examples-linux-imx.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build example - i.MX Linux
+
+on:
+    push:
+    pull_request:
+
+concurrency:
+    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+    imx:
+        name: Linux i.MX Build
+        timeout-minutes: 70
+
+        runs-on: ubuntu-latest
+        if: github.actor != 'restyled-io[bot]'
+
+        container:
+            image: connectedhomeip/chip-build-imx:0.5.52
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+              with:
+                  submodules: true
+
+            - name: Build App
+              timeout-minutes: 10
+              run: |
+                  ./scripts/examples/imxlinux_example.sh \
```

Comment:
can we add `build_examples.py` support instead of separate scripts?

Reply:
Thank you for the comments.
Actually that I'm afraid that so far we cannot make it in short term. Looks build_examples.py will use the predefined target in /build folder but i.MX platform don't have such target.
Also You can see the imxlinux_example.sh, there are some macro override into the build parameters. So for the initial CI, we'd like to keep use the script to build.
But long tern, we will try to setup the i.MX build target and then align with other platform to use build_examples.py.

Reply:
build_examples creates its own folders by names, so it can be any fake name, likely as long as it has imx in the name.

What I am trying to ensure is that the builds have a away to specify output directory and do not pollute the source tree.

Reply:
Thank you for the comments. 
The script for imx actually support specify output directory. You can see that the imxlinux_example.sh need 2 arguments. The second one "_examples/chip-tool/out/aarch64_" can be replaced by any specify output target and in my test it can be put into ${MATTER}/out/aarch64 which is the same output folder of build_examples.py. It is quite similar like another NXP platform build script k32w_se_example.sh.
The _build_examples.py_ is a great tool in matter and it support multiple arguments and I'll try to align it with imx builds long term.

Reply:
what does "long term" mean in this context? Will it be done before v1 branch (i.e. within 1 month)?

Reply:
Thank you for the comments.
For your question, I'm apologize that we cannot make it happen in 1 month. Since so far the imxlinux_example.sh can help us to improve the lengthy doc for imx platform, so we want to merge this version first. And then we refine the document to make developer easier to deploy matter in their imx platforms. 
Means for this 1 month, we'd like to refine the document, add the vscode-image build command with imx platform and validate all available examples on imx platforms with the docker images(submit platform dependency fix if need).
So that please help to merge this version first.